VVV cardfiles for Run 3 MC production

maravin opened the pull request:
Added cardfiles for four processes for the Run 3 MC production. These are exact copies of the UL processes with the only changes of the cms energy (6500 -> 6800).

maravin commented:
Ping :)

menglu21 commented on bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/WWZJets_4f_NLO/WWZJets_4f_NLO_proc_card.dat:
```diff
@@ -0,0 +1,13 @@
+
+# exclude s-channel top contribution to avoid
+# overlap with TTZ sample
+generate p p > w w z $$ t t~ [QCD] @0
```
since you are using 4 flavor, there should be no overlap with ttz. another question is about the w/z decay, you will decay them through pythia, right

maravin replied:
Correct!

Saptaparna replied:
The W's with this syntax, like Meng said, will be produced on-shell with a delta function in place of the actual width. Using madspin for decays could be a good idea.

maravin replied:
I can ask the SMP conveners if they want to do that. This is a repeat of the samples from 13 TeV and we have not received any recommendations or requests for a change.

menglu21 commented on bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/WZGToLNu2jG_4f_NLO/WZGToLNu2jG_4f_NLO_proc_card.dat:
```diff
@@ -0,0 +1,40 @@
+#************************************************************
+#*                     MadGraph5_aMC@NLO                    *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 2.4.2                 2016-06-10         *
+#*                                                          *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph5_aMC@NLO         *
+#*                                                          *
+#*     run as ./bin/mg5_aMC  filename                       *
+#*                                                          *
+#************************************************************
+set loop_optimized_output True
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
```
the banner can be removed

maravin replied:
Done.

menglu21 commented on bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/WZGToLNu2jG_4f_NLO/WZGToLNu2jG_4f_NLO_run_card.dat~:
```diff
@@ -0,0 +1,147 @@
+# At least all massless quarks of the model should be included here.   *
+#***********************************************************************
+ 4 = maxjetflavor
+#***********************************************************************
```
this file should be removed, and you should have a madspin card for Z > j j

maravin replied:
Done.

menglu21 commented on bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/WZZJets_5f_NLO/WZZJets_5f_NLO_madspin_card.dat~:
```diff
@@ -0,0 +1,30 @@
+decay z > all all
+
+# running the actual code
+launch
```
this file should be removed

maravin replied:
Done.

menglu21 commented on bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/WZZJets_5f_NLO/WZZJets_5f_NLO_proc_card.dat:
```diff
@@ -0,0 +1,11 @@
+define j = j b b~
+define w = w+ w-
+
+generate p p > w z z [QCD] @0
```
this syntax will include WH diagram, if you don't want it you should add "$$ h"

menglu21 commented on bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/WZZJets_5f_NLO/WZZJets_5f_NLO_madspin_card.dat:
```diff
@@ -0,0 +1,30 @@
+#************************************************************
+#*                        MadSpin                           *
+#*                                                          *
+#*    P. Artoisenet, R. Frederix, R. Rietkerk, O. Mattelaer * 
+#*                                                          *
+#*    Part of the MadGraph5_aMC@NLO Framework:              *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+#
+#directory for gridpack mode
```
please remove the banner

menglu21 commented on bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ZZZJets_5f_NLO/ZZZJets_5f_NLO_proc_card.dat:
```diff
@@ -0,0 +1,10 @@
+define p = p b b~
+define j = j b b~
+
+generate p p > z z z [QCD] @0
```
this syntax will include ZH diagram, if you don't want it you should add "$$ h"

menglu21 reviewed (commented):
besides the comments for each file, a general comment is that you plan to have inclusive Matrix-element and decay them in pythia for different decay mode. One con is that you may loss contribution from off-shell, e.g., for ZZZ to 2L/4L channel, the difference between process "z z z" and "z z lep+ lep-" may not be negligible, maybe you should take those into consideration

maravin commented:
I have removed *~ files and removed the banners. I have also pinged SMP conveners to weigh on the issue with the MadSpin and off-shell bosons, I will post an update here as soon as I hear from them. Thank you!

amecca commented on bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/WWZJets_4f_NLO/WWZJets_4f_NLO_run_card.dat:
```diff
@@ -0,0 +1,146 @@
+# Photon-isolation cuts, according to hep-ph/9801442                   *
+# When ptgmin=0, all the other parameters are ignored                  *
+#***********************************************************************
+  20  = ptgmin    ! Min photon transverse momentum
```
```suggestion
  15  = ptgmin    ! Min photon transverse momentum
```
We used this pt cut in [the request](https://github.com/cms-sw/genproductions/pull/3172) for WZG/ZZG --> 2l 2j G

maravin commented:
Hello, I heard back from SMP/HZZ/Higgs folks and the consensus is to replicate the Run 2 UL releases exactly with an exception of a proper PDF and new center of mass energy, so the cardfiles should stay as they are. There is an expectation to modify these cardfiles to allow off-shell and use MadSpin for decays, but they will be done at a later stage if necessary.

menglu21 commented on bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/WZGToLNu2jG_4f_NLO/WZGToLNu2jG_4f_NLO_proc_card.dat:
```diff
@@ -0,0 +1,18 @@
+define j = g u c d s u~ c~ d~ s~
+define lep = e+ mu+ ta+ e- mu- ta-
+define nu = ve vm vt ve~ vm~ vt~
+generate p p > lep nu z a [QCD] @0
```
I may overlook this line previously, you are using "LNu2jG" which means you specify the Z>2j, maybe you could modify the name or decay the Z boson here to make them more consistent

maravin replied:
Done!
